Governance.md

```diff
@@ -146,29 +206,28 @@ The process of making technical or code contributions is the same for everyone,
   - Review the Project (https://github.com/BOINC/boinc/projects) associated with the area of BOINC in which you want to contribute
   - Issues that have been reviewed and are ready for implementation are listed under Longterm or TODO
   - Issues with a higher priority for implementation are listed under TODO
-- Follow the software development process that BOINC uses (See [Development_Workflow.md](../../blob/master/Development_Workflow.md))
+- Follow the software development process that BOINC uses (See [Development_Workflow.md](Development_Workflow.md))
 
-If you are reporting a bug or requesting a feature, make sure you review the [Development Workflow](../../blob/master/Development_Workflow.md) before you submit it.
+If you are reporting a bug or requesting a feature, make sure you review the [Development Workflow](Development_Workflow.md) before you submit it.
 
 ## 5. Decision processes
 ### 5.1 Voting Processes
 Because one of the fundamental aspects of accomplishing things within the BOINC framework is doing so by consensus, it is necessary to determine whether consensus has been reached. This is done by voting.
 
 There are a few types of items that require a vote:
-
 - Whether or not to fix a bug or implement a feature request (documented and voted on as an issue on github)
 - The design of a proposed feature or bug fix (documented and voted on within the relevant issue on github)
 - A change in code or configuration to the system (documented and voted on as a pull request on github)
 - General availability of stable releases (documented and voted on in the boinc_alpha mailing list)
 - Procedural and other issues
```

Comment:
I find this "procedural and other issues" list quite confusing:

- "Other committer votes not otherwise identified above"
  - I guess this maybe means "Other committer votes (as outlined in 5.2 below) not otherwise identified above"?
  - But [a] there don't really seem to be any such topics?  And [b] it's awkward to have two kind-of-sort-of-the-same lists of the domain of committer decisions.
- "Other PMC votes not otherwise identified above"
  - There are no PMC votes in the list above?

-- So end result is that I find this really unclear.

Reply:
Related: 
- What is meant by "procedural", and in the last line of section 5.1 ("Procedural and other issues shall always follow the majority voting process.")

- I assume all topics not specifically delegated to Committers should be retained by the PMC

Reply:
Related: 
- In the last sentence of section 5.2 it says "Since most decisions that committers are involved in will use consensus voting, it is important for them to try to remain aware of proposed items." -- is the idea that there exists a category of decision for Committers, "Procedural", which use majority votes?   If this is the intent, it's unclear (to me) what sorts of decisions are in that category -- in particular, as distinct from issues of Governance (which is the domain of the PMC).

Reply:
I've significantly modified section 5 to try to clarify things - please review and provide additional feedback.

Reply:
In 5.2 there is a list of bullets; the last few are confusing; i'm not clear what the intension is here:

>* ....
>* All other committer decisions should be discussed and voted on in the boinc_dev mailing list.
>* Procedural and other issues
>    * Other committer votes not otherwise identified will be voted on in the boinc_dev mailing list
>    * Other PMC votes not otherwise identified will be voted on in the boinc_adm or private boinc_pmc mailing lists as deemed appropriate by the PMC.

Seems here we specify three different rules to handle "votes not otherwise identified"?  I'm not clear what differences are?  (*Maybe one of these bullets, e.g. the one that says "other PMC votes", was meant to be somewhere else?)

I'm not sure, but I think the idea of the "procedural and other issues" sub-bullets is to establish that in the event it's unclear whether an issue is a committer decision, or if it's unclear where the committer decision should be discussed, then the PMC would determine this (in bonc_adm or boinc_pmc as deemed appropriate by the PMC)?

Reply:
Any thoughts on this?  (*Bullet-list in 5.2)

-- Don't see any changes, nor comments to suggest you disagree with my confusion?  But seems somewhat important?

Reply:
(*All other changes in this PR9 look ok to me)

Reply:
Matt - can you look at the text itself again?  https://github.com/BOINC/boinc-policy/blob/knr_supporters/Governance.md#52-committer-decisions   I revised it so that the bullet points specify where discussions occur and the second set of points specifies the rules that apply.